components/GroupSearchFilters/GroupSearchFilters.styles.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Box, Button } from 'ui-kit';
+
+const GroupSearchFilters = {};
+
+const Divider = (props = {}) => (
+  <Box
+    as="hr"
+    mt="s"
+    mb="base"
+    border="none"
+    bg="neutrals.200"
+    height={2}
+    {...props}
+  />
+);
+
+function FilterButton(props = {}) {
+  return (
+    <Button
+      mr="s"
+      onClick={props.onClick}
+      rounded={true}
+      size="s"
+      variant="chip"
+      {...props}
+    >
+      {props.label}
+      {typeof props.labelDetail === 'number' ? (
+        <Box
+          as="span"
+          bg="primary"
+          borderRadius="xl"
+          color="paper"
+          ml="xs"
+          px="xs"
+        >
+          {props.labelDetail}
+        </Box>
+      ) : (
+        <Box as="span" color="primary" ml="xs">
+          {props.labelDetail}
+        </Box>
+      )}
+    </Button>
+  );
+}
+
+FilterButton.propTypes = {
+  label: PropTypes.string,
+  labelDetail: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  onClick: PropTypes.func,
+};
+
+GroupSearchFilters.Divider = Divider;
+GroupSearchFilters.FilterButton = FilterButton;
+
+export default GroupSearchFilters;
```

Comment:
The `[FILE].styles.js` is just for Styled Components, so we could either include these as subcomponents in `GroupSearchFilters`, or create a new `GroupSearchFilters/[ComponentName].js` and have that in there.

Reply:
That makes sense, this didn't feel right but in lieu of a clear easy-to-find example of a component having "child" or sub-components — which I now see there actually is 🤦‍♂️ — I figured this would get caught in PR review.

I'll gladly make this change.

Reply:
Fixed in f21b4c7. Also realized we already had a `<Divider>` component in `ui-kit` (again, 🤦‍♂️) so used it instead.